hz.ps1

```diff
@@ -697,6 +697,29 @@ function ensure-server-files {
             }
         }
 
+        if (-not $found) {
```

Comment:
I *think* this may have been fixed already in the recent changes to the `hz.ps1` script. If you specify `-server-version master` it *does* find the config file. If it does not work, can you give me a server version that does not work (`5.1-SNAPSHOT`? I think even that one works now). I am not too sure I want to allow falling back to the previous minor... and in any case, if `$minor` is zero, we cannot go to `-1` :) So... I'd rather first try to see if it works, with the current script.